app/assets/javascripts/application.js

```diff
@@ -14,3 +14,5 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+
+//= require uikit
```

Comment:
Szerintem ezt inkabb tedd az elozo csoportba a `require_tree .` ele, mert kulonben erdekes dolgok tortenhetnek, mert a UIKit js file-ok kesobb toltodnek be mint a te js file-jaid.